src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoFeetPlace.java

```diff
@@ -0,0 +1,255 @@
+package me.zeroeightsix.kami.module.modules.combat;
+
+import me.zeroeightsix.kami.command.Command;
+import me.zeroeightsix.kami.module.Module;
+import me.zeroeightsix.kami.module.ModuleManager;
+import me.zeroeightsix.kami.setting.Setting;
+import me.zeroeightsix.kami.setting.Settings;
+import me.zeroeightsix.kami.util.BlockInteractionHelper;
+import me.zeroeightsix.kami.util.Wrapper;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockObsidian;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.item.EntityItem;
+import net.minecraft.entity.item.EntityXPOrb;
+import net.minecraft.item.ItemBlock;
+import net.minecraft.item.ItemStack;
+import net.minecraft.network.play.client.CPacketEntityAction;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Vec3d;
+
+import static me.zeroeightsix.kami.util.BlockInteractionHelper.*;
+
+/**
+ * Created 13 August 2019 by hub
+ * Updated 21 November 2019 by hub
+ */
+@Module.Info(name = "AutoFeetPlace", category = Module.Category.COMBAT)
+public class AutoFeetPlace extends Module {
+
+    private final Vec3d[] surroundTargets = {
+            new Vec3d(1, 0, 0),
+            new Vec3d(0, 0, 1),
+            new Vec3d(-1, 0, 0),
+            new Vec3d(0, 0, -1),
+            new Vec3d(1, -1, 0),
+            new Vec3d(0, -1, 1),
+            new Vec3d(-1, -1, 0),
+            new Vec3d(0, -1, -1),
+            new Vec3d(0, -1, 0)
+    };
+
+    private Setting<Boolean> triggerable = register(Settings.b("Triggerable", true));
+    private Setting<Integer> triggerableTimeoutTicks = register(Settings.i("Triggerable Timeout (Ticks)", 20));
+    private Setting<Integer> blockPerTick = register(Settings.i("Blocks per Tick", 4));
```

Comment:
Placing 4 blocks in a tick probably doesn't play well with anticheats, does it?
Also, wouldn't this settings be easier to understand if it:

- Was ticks per block instead of blocks per tick
- Had minimum and maximum values

Reply:
Placing multiple blocks per tick works suprisingly well on the servers i tested on, at a rate of 4 per tick, i never had ghostblock issues. When placing a larger amount of blocks tho, some amount of them gonna be clientside glitchblocks.

Min and max values make perfect sense tho, gonna add them. Im just not sure what the max would be here. I guess i would go with 9 as in the maximum blocks needed to be safe.
This however implies that the user knows about glitchblocks and does not just max the setting.

Reply:
Added min max values for [blocksPerTick ](https://github.com/zeroeightysix/KAMI/pull/141/commits/5f6f12b0fb0c1ba66ca5e66194cbea93c0c2747c) and [timeoutTicks](https://github.com/zeroeightysix/KAMI/pull/141/commits/fd42105f7f05261073145a0030659fb1472d6e61).

Reply:
After some research, it seems that ticks per block seems to be the way to go here. It causes desync for some people, other then was i was experiencing while testing.